Commands.adoc

```diff
@@ -0,0 +1,226 @@
+   export DOCKER_TLS_VERIFY=1
+   export DOCKER_API_VERSION=1.21
+
+   # openshift env:
+   # You can access the OpenShift console on: https://10.1.2.2:8443/console
+   # To use OpenShift CLI, run: oc login https://10.1.2.2:8443
+   export OPENSHIFT_URL=https://10.1.2.2:8443
+   export OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console
+
+   export DOCKER_REGISTRY=hub.openshift.centos7-adb.10.1.2.2.xip.io
+
+   # run following command to configure your shell:
+   # eval "$(vagrant service-manager env)"
+-----------------------------------------------------------------------------
+
+--
+
+. If you need to run the plugin and get the environment variables and
+certificates for a specific provider use: +
+For Docker:
++
+--
+----------------------------------------------------------------------------
+  $ vagrant service-manager env docker
+  # Set the following environment variables to enable access to the
+  # docker daemon running inside of the vagrant virtual machine:
+  export DOCKER_HOST=tcp://172.28.128.3:2376
+  export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+  export DOCKER_TLS_VERIFY=1
+  export DOCKER_API_VERSION=1.21
+
+  # run following command to configure your shell:
+  # eval "$(vagrant service-manager env docker)"
+----------------------------------------------------------------------------
+
+--
+*Note:* In the case of the docker service, the required TLS certificates for
+securing the Docker communication will be regenerated and copied to the host
+as part of the command execution. If existing certificates are found to be
+invalid, they get regenerated.
++
+--
+Similarly for OpenShift you can use: +
+`$ vagrant service-manager env openshift`
+
+--
+. To run the plugin and get the environment variables and certificates for
+docker in key=value format use:
++
+--
+----------------------------------------------------------------------
+$ vagrant service-manager env docker --script-readable
+DOCKER_HOST=tcp://172.28.128.3:2376
+DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker'
+DOCKER_TLS_VERIFY=1
+DOCKER_API_VERSION=1.21
+----------------------------------------------------------------------
+--
+
+== Identify the IP Address and Version of the Running VM
+
+If you need to find information related to the running VM, such as the routable
+IP address or the version and release information use:
+
+`$ vagrant service-manager box` +
+
+Usage: `vagrant service-manager box [sub-command] [options]``
+
+The following sub-commands can be used: +
+`ip`: Displays the routable IP address of the running VM +
+`version`: Displays the version and release information about the running VM +
+
+The following options can be used: +
+`--script-readable` : Displays information in a key=value format +
+`-h, --help, print this help` : Prints help for the specific command being run.
+
+=== Examples
+
+. `$ vagrant service-manager box ip +
```

Comment:
nope, there is an issue. That does not render as expected. This becomes one lines. As do the other example commands. The syntax is wrong here. Same for the other examples in this section.

Also why a numbered list. Implies some order which is not the case imo.